src/klein/test/test_headers.py

```diff
@@ -188,12 +189,13 @@ def test_pairValueText(self, value: str) -> None:
         self.assertEqual(normalized, ((b"name", headerValueAsBytes(value)),))
 
 
-class GetValuesTestsMixIn(object):
+class GetValuesTestsMixIn(ABC):
```

Comment:
Use an ABC and mark methods abstract instead of raising when called; this catches code errors immediately and removes untested code.